### PR TITLE
Added Deployment settings

### PR DESCRIPTION
**Pull request by anurag-ks**
This is for my task - https://www.google-melange.com/gci/task/view/google/gci2014/6388032466845696

### CODE REVIEW

File: modules/s3cfg.py [hunk 1]
```diff
@@ -2078,6 +2078,13 @@ def get_cr_shelter_housing_unit_management(self):
         """
         return self.cr.get("shelter_housing_unit_management", False)
 
+    def get_cr_shelter_notification_subject(self):
+        """
+            Subject for the Shelter Notification Dispatcher
+        """
+        T = current.T
```
Comment (hitesh96db) [inline modules/s3cfg.py]: Put this T in the beginning of the file ?( no need to define it everytime then. )

**Comment by anurag-ks** [comment]
Done the change, If it's okay please mark my task thanks :+1:

File: modules/s3cfg.py [hunk 2]
```diff
@@ -2078,6 +2080,12 @@ def get_cr_shelter_housing_unit_management(self):
         """
         return self.cr.get("shelter_housing_unit_management", False)
 
+    def get_cr_shelter_notification_subject(self):
+        """
+            Subject for the Shelter Notification Dispatcher
+        """
+        return self.cr.get("notification_subject", T("Deployment Request"))
```
Comment (hitesh96db) [inline modules/s3cfg.py]: Sorry, I should've pointed these mistakes out in one go :/
cr.get("shelter_notification_subject"..... ?

File: private/templates/default/config.py [hunk 3]
```diff
@@ -446,6 +446,8 @@
 #settings.cr.shelter_population_dynamic = True
 # Uncomment to disable people registration in shelters
 #settings.cr.people_registration = False
+# Uncomment to change the notification subject
+#settings.cr.notification_subject = "Deployment Requests"
```
Comment (hitesh96db) [inline private/templates/default/config.py]: shelter...

**Comment by anurag-ks** [comment]
thanks

**Pull request merged**